android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -21,6 +21,20 @@
 import android.support.v4.app.NotificationCompat;
 import android.util.Log;
 
+import android.support.annotation.Nullable;
```

Comment:
Suggest to use the androidx implementation

import androidx.annotation.NonNull;